e2e/angular-core/src/projects.test.ts

```diff
@@ -362,4 +363,39 @@ describe('Angular Projects', () => {
     // ASSERT
     expect(buildOutput).toContain('Successfully ran target build');
   }, 300000);
+
+  it('Custom Webpack Config for SSR - should serve the app correctly', async () => {
+    // ARRANGE
+    const ssrApp = uniq('app');
+
+    runCLI(`generate @nrwl/angular:app ${ssrApp} --no-interactive`);
+    runCLI(`generate @nrwl/angular:setup-ssr ${ssrApp} --no-interactive`);
+
+    updateJson(`apps/${ssrApp}/project.json`, (project) => {
+      project.targets.server.executor = '@nrwl/angular:webpack-server';
+      return project;
+    });
```

Comment:
When it comes to the workspace/project configurations, better to use the utilities that deal with them. It abstracts things like the filenames and provide type safety.
```suggestion
    updateProjectConfig(ssrApp, (project) => {
      project.targets.server.executor = '@nrwl/angular:webpack-server';
      return project;
    });
```

Reply:
Ah I wasn't aware there was a utility in the e2es for this, I'll make this change